running import.py
Parsing columns with \n due to error in original file
